docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Density
+
+`DataGrid` provides an option to set the density of the rows and the columns header.
+
+### Density prop
+
+There are three possible density options to choose from: `small`, `medium` and `large`. By default the density of the `DataGrid` is set to `medium`. The default grid density can be changed by setting the `density` propery. The `density` propery works with relation to the `rowHeight` and `headerHeight` properties. This means that it will respect the values provided using `rowHeight` or `headerHeight` properties.
```

Comment:
```suggestion
There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the `DataGrid` is set to `medium`. The default grid density can be changed by setting the `density` prop. The `density` prop works with relation to the `rowHeight` and `headerHeight` properties. This means that it will respect the values provided using `rowHeight` or `headerHeight` props.
```

Reply:
Based off @oliviertassinari's:
```suggestion
There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the `DataGrid` is set to `medium`. The default grid density can be changed by applying the `density` prop. The `density` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
```